jlm/llvm/opt/alias-analyses/Andersen.cpp

```diff
@@ -12,6 +12,18 @@
 namespace jlm::llvm::aa
 {
 
+/**
+ * If values of the given type may point to things, the set of pointees needs to be tracked.
+ * If the type is an aggregate type, pointees must be tracked if any subtype is pointing.
+ * @param type the rvsdg type to be checked
+ * @return true if pointees should be tracked for all values of the given type, otherwise false
+ */
+bool
+IsTypePointing(const rvsdg::type & type)
```

Comment:
Do you think a better name would be: `IsOrContainsPointerType()`? I had to read the documentation multiple times before I fully understood how the code in the body relates to the name.

Reply:
That would be a better name, I'll change it